bin/sam-translate.py

```diff
@@ -1,5 +1,3 @@
-#!/usr/bin/env python2
```

Comment:
We still want to keep this and update from `python2` to `python` as this file is executable.

Reply:
Added back as `python`.